### ISSUE TITLE
electron.remote.screen.getAllDisplays is not a function (telemetry.js:77)

### ISSUE THREAD

**Issue by feross** [webtorrent/webtorrent-desktop#1896]
I've noticed that with `electron@11` this error is thrown 
```
electron.remote.screen.getAllDisplays is not a function (telemetry.js:77)
```

I've tested and this error does not appear with `electron@10`, so I suspect that could be something related with the warning that refers to https://github.com/electron/electron/issues/23506

I'm trying to find a fix, but I've not find a solution yet 😓

_Originally posted by @hicom150 in https://github.com/webtorrent/webtorrent-desktop/issues/1895#issuecomment-729677773_

**Comment by feross** [comment]
I think it might be related to https://github.com/electron/electron/issues/21408 which has to do with a change to the `remote` module.

**Comment by andersk** [comment]
Filed upstream as electron/electron#26610.

**Comment by feross** [comment]
@andersk Thank you for filing an upstream bug!

**Comment by feross** [comment]
Seems like the issue was fixed on `master`. Once it's in a release, we can update Electron and see if this issue is fixed.

**Comment by andersk** [comment]
My fix was backported in 11.0.3, just released.

**Comment by feross** [comment]
@andersk Great, thank you for getting this fixed! I just updated Electron on `master` and confirmed that the issue has been fixed. 🤩
